projects/agent-framework-javascript/agent-framework-javascript.md

```diff
@@ -0,0 +1,97 @@
+# Project Name
+
+This is still up for debate. Until then, the working title is _Agent Framework JavaScript_.
+
+# Preferred Maturity Level
+
+Considering the fact that the project has been adopted by various companies and governments, and powers a number of production use cases, the _impact_ stage seems appropriate. However, the project does not fulfill the "is commonly used in enterprise production environments".
+
+Therefore, the project is estimated to be in the _growth_ stage, approaching the _impact_ stage.
+
+# Project Description
+
+Aries Framework JavaScript has evolved significantly since its inception as a Hyperledger Aries project. Initially, it heavily relied on Hyperledger standards such as DIDComm, Indy, and AnonCreds. However, with advancements in verifiable credential technology and the emergence of new standards, the framework underwent multiple refactoring and modularization processes to maintain interoperability.
+
+This allowed for the inclusion of non-Hyperledger standards like W3C Verifiable Credentials with Data Integrity Proofs, Presentation Exchange, OpenID4VC, and SD-JWT integration. As industry requirements shifted towards greater modularity, it became apparent that a unified framework may be better.
+
+The future direction for Aries Framework JavaScript involves adopting a compartmentalized approach consisting of single-purpose libraries designed to work together seamlessly, building on what is already out there. This transition will take considerable effort and will, therefore, be gradual.
+
+In order to expand the framework's support for standards beyond the Hyperledger ecosystem, a reassessment of its governance was necessary. The Open Wallet Foundation (OWF) has been considered a potential steward due to its commitment to promoting interoperability without directly developing or maintaining standard protocols.
+
+![afj-high-level-arch.png](./afj-high-level-arch.png)
+
+> **NOTE<sup>1<sup>:** In addition to the [Aries Framework JavaScript repository](https://github.com/hyperledger/aries-framework-javascript), we propose moving the [Aries Framework JavaScript Extensions](https://github.com/hyperledger/aries-framework-javascript-ext) repository too, due to its tight relationship with the framework.
+>
+> **NOTE<sup>2<sup>:** Another Hyperledger project that is tightly related to the framework is [Aries Mobile Agent React Native (a.k.a. Bifold)](https://github.com/hyperledger/aries-mobile-agent-react-native). This project is maintained by a different set of people and is therefore not included in this proposal.
```

Comment:
What is the direction of the relationship with the [Bifold](https://github.com/hyperledger/aries-mobile-agent-react-native). Does the proposed project depend on Bifold or is the other way around?

Reply:
Bifold depends of this project (it is an application built on the framework).